cross/acl/Makefile

```diff
@@ -0,0 +1,21 @@
+PKG_NAME = acl
+PKG_VERS = 2.2.52
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).src.$(PKG_EXT)
+PKG_DIST_SITE = http://download.savannah.gnu.org/releases/acl
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/attr
+
+HOMEPAGE = http://kernel.org/
+COMMENT  = 
+LICENSE  = 
```

Comment:
should be GPLv2+ as of https://savannah.nongnu.org/projects/acl
Note: might add the Comment there as well ;)

Reply:
Ok, I will update License and add Comment for acl package